applications/crossbar/doc/storage.md

```diff
@@ -223,7 +223,77 @@ Key | Description | Type | Default | Required | Support Level
 `attachments` |   | [#/definitions/storage.plan.database.attachment](#storageplan.database.attachment) |   | `false` |  
 `connection` |   | `string()` |   | `false` |  
 
+#### Create
 
+First, we need a 32-character UUID which will be the ID of the S3 configuration that we'll use later in the storage plan setup. Creating a UUID is easy to do with the following cURL command:
+
+```shell
+curl https://www.uuidgenerator.net/api/version4/1 | sed 's/-//g'
+e3e26e06a4b1465599e6dc9e1516fd8b
+```
```

Comment:
can we remove the reference to `https://www.uuidgenerator.net` and add instead  `echo $(tr -dc a-f0-9 < /dev/urandom | dd bs=32 count=1 2> /dev/null)`